assets/js/translator.js

```diff
@@ -749,50 +753,54 @@ function detectLanguage() {
         textTranslateRequest.abort();
     }
 
-    textTranslateRequest = $.jsonp({
```

Comment:
Looks like we're missing a newline before this function, would you mind adding that in?